ocfweb/docs/views/officers.py

```diff
@@ -106,19 +103,21 @@ def officer_from_uid(uid):
     Term('Summer 2013', gms=['daradib'], sms=['tzhu']),
     Term('Fall 2013', gms=['daradib'], sms=['tzhu']),
     Term('Spring 2014', gms=['nickimp'], sms=['ckuehl']),
+    Term('Summer 2014', gms=['nickimp'], sms=['ckuehl']),
     Term('Fall 2014', gms=['nickimp'], sms=['ckuehl']),
     Term('Spring 2015', gms=['nickimp'], sms=['ckuehl']),
+    Term('Summer 2015', gms=['nickimp'], sms=['ckuehl']),
```

Comment:
(i actually don't care though, if you think it's silly to list summer I'm fine with removing them)